### ISSUE TITLE
c3d_sport1m_feature_extraction_frm.sh  error

### ISSUE THREAD

**Issue by mengliu1991** [facebook/C3D#357]
Hi,
 I run the test code using sh c3d_sport1m_feature_extraction_frm.sh, it output the following errors. Do you know the reason? Thank you.

I0211 20:22:02.938414 11594 net.cpp:154] accuracy needs backward computation.
I0211 20:22:02.938419 11594 net.cpp:165] This network produces output accuracy
I0211 20:22:02.938441 11594 net.cpp:183] Collecting Learning Rate and Weight Decay.
I0211 20:22:02.938459 11594 net.cpp:176] Network initialization done.
I0211 20:22:02.938467 11594 net.cpp:177] Memory required for Data 5653289408
F0211 20:22:02.938547 11594 io.cpp:56] Check failed: fd != -1 (-1 vs. -1) File not found: conv3d_deepnetA_sport1m_iter_1900000
*** Check failure stack trace: ***
c3d_sport1m_feature_extraction_frm.sh: line 7: 11594 Aborted                 (core dumped) GLOG_logtosterr=1 ../../build/tools/extract_image_features.bin prototxt/c3d_sport1m_feature_extractor_frm.prototxt conv3d_deepnetA_sport1m_iter_1900000 -1 50 1 prototxt/output_list_prefix.txt fc7-1 fc6-1 prob

**Comment by dutran** [comment]
It seems you did not downloand the pre-trained models, (e.g. conv3d_deepnetA_sport1m_iter_1900000)
